partiql-eval/src/main/kotlin/org/partiql/eval/internal/operator/rel/RelOpExclude.kt

```diff
@@ -173,7 +172,7 @@ internal class RelOpExclude(
             // apply exclusions to subtree
             var value = element
             // apply collection index exclusions at deeper levels for lists and sexps
```

Comment:
Can remove reference to `sexps` here

```suggestion
            // apply collection index exclusions at deeper levels for lists
```